src/azure-cli/azure/cli/command_modules/appconfig/commands.py

```diff
@@ -31,6 +33,18 @@ def load_command_table(self, _):
         client_factory=cf_configstore_operations
     )
 
+    configstore_feature_util = CliCommandType(
```

Comment:
From CLI code convention, we will treat the operations you defined in your source code as custom command type, and the commands using self-defined method as custom_command. For more information, you can see[ here](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands). Example usage is [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/commands.py#L33) and [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/commands.py#L66)